merkle/tile/tiles.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package tile includes functionality for describing a Merkle tree in
+// terms of tiles and coordinates.
+package tile
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// Coords represents the location of a single tile from a cover of a Merkle tree.
```

Comment:
Might help to define "a cover" here? Also further down "encompasses" is used. Possibly keeping the same terms will help readers?